plumbing/format/packfile/delta_selector.go

```diff
@@ -0,0 +1,176 @@
+package packfile
+
+import (
+	"sort"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/plumbing/storer"
+)
+
+const (
+	// deltas based on deltas, how much steps we can do
+	maxDepth = int64(50)
+)
+
+// set of object types that we should apply deltas
+var applyDelta = map[plumbing.ObjectType]bool{
```

Comment:
"set of object types that we should apply deltas" -> "applyDelta is the set of object types that we should apply deltas"